commercetools-models/src/test/java/io/sphere/sdk/orders/commands/OrderFromCartCreateCommandIntegrationTest.java

```diff
@@ -82,4 +85,16 @@ public void shouldChangeShippingMethodStateWithNonMatchingShippingMethod()  thro
             });
         });
     }
+
+
+    @Test
+    public void orderFromCartDraft(){
+        withFilledCart(client(), cart -> {
+            final OrderFromCartDraft orderFromCartDraft = OrderFromCartDraftBuilder.of(cart.getId(),cart.getVersion() ).shipmentState(ShipmentState.SHIPPED).build();
+            final Order order = client().executeBlocking(OrderFromCartCreateCommand.of(cart).withDraft(orderFromCartDraft));
+            assertThat(order).isNotNull();
+            assertThat(order.getShipmentState()).isEqualByComparingTo(ShipmentState.SHIPPED);
```

Comment:
Just a question: Why do we have to use `isEqualByComparingTo` here? Does `isEqualTo` not work?

Reply:
Didn't think about it, just used it 😅 , but for conformity i would change it to `isEqualTo`.